### PR TITLE
add more tox tests options to the docs

### LINKED ISSUE #108: Update readme with instructions for running tests
From @mr-c on Matrix, 

```bash
tox -e py39-pydocstyle
make diff_pydocstyle_report
make install-deps
```

I think this can go to the `README.md`?

### PR DESCRIPTION
**Pull request by Th3nn3ss**
Closes #108

- add more test options for tox in the `README`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ pip install -e .
 make install-dep
 make test  # just the unit tests
 tox  # all the code checks
+tox -e py39-pydocstyle # perform only pydocstyle tests (py39 is the version of the python interpreter you have installed)
```
Comment (mr-c) [inline README.md:64]: I would suggest a section on how to run a specific tox environment using this as an example. And how to list all of the configured tox environments (`tox -l` if I remember correctly)

@kinow Later you and I should synchronize a CONTRIBUTING.md across all the CWL code projects. I'm rather partial to one I worked on for k-h-mer: https://khmer.readthedocs.io/en/latest/dev/getting-started.html ( from https://khmer.readthedocs.io/en/latest/dev/ )

**Review by mr-c — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -61,6 +61,8 @@ pip install -e .
...
+make diff_pydocstyle_report # run a diff to show how much changes where made in the docstryle
```
Comment (mr-c) [inline README.md:65]: Maybe mention `make help` to list the major Makefile targets?

File: README.md (hunk 1 above)
Comment (kinow) [reply README.md:64]: >I would suggest a section on how to run a specific tox environment using this as an example. And how to list all of the configured tox environments (tox -l if I remember correctly)

+1

>@kinow Later you and I should synchronize a CONTRIBUTING.md across all the CWL code projects. I'm rather partial to one I worked on for k-h-mer: https://khmer.readthedocs.io/en/latest/dev/getting-started.html ( from https://khmer.readthedocs.io/en/latest/dev/ )

Great idea! During the video call I was actually looking for a CONTRIBUTING.md, before suggesting the README.md update. So I guess other developers would probably expect this useful building instructions to be somewhere in a CONTRIBUTING file.

And great job on the khmer file! It even includes multi-platform docs :clap: Let's do it!

File: README.md (hunk 2 above)
Comment (kinow) [reply README.md:65]: And s/docstryle/docstyle

**Review by kinow — COMMENTED** [review]
Looking great @Th3nn3ss ! Just needs to address @mr-c 's feedback and then this should be good to be merged. That was quick! Thanks.

File: README.md (hunk 1 above)
Comment (kinow) [reply README.md:67]: @Th3nn3ss just pending to address this part of @mr-c 's feedback

>I would suggest a section on how to run a specific tox environment using this as an example. And how to list all of the configured tox environments (tox -l if I remember correctly)

Then we should be ready to merge it :tada:

**Pull request merged**